src/nupic/engine/Network.hpp

```diff
@@ -376,6 +377,11 @@ namespace nupic
     static void
     registerRegionPackage(const char * path);
 
+    /*
+     * Adds a cpp region to the RegionImplFactory's packages
+     */
+    static void registerCPPRegion(const std::string name, GenericRegisteredRegionImpl* wrapper);
+
```

Comment:
this is for registering a Python region? It isn't obvious from the name or comment